### PR TITLE
Fix tables in tabs + who_has_perm issue 

### PR DESCRIPTION
**Pull request by a-belhadj**

### CODE REVIEW

File: profiles/api/serializers/scope_serializer.py [hunk 1]
```diff
@@ -15,10 +15,30 @@ class Meta:
...
+        else:
+            return ""
+
+    def get_team(self, obj):
+        if obj.is_org:
+            return {}
+        elif obj.is_team:
+            return {"id": obj.id, "name": obj.name}
+        else:
+            return ""
```
Comment (Sispheor) [inline profiles/api/serializers/scope_serializer.py:40]: return dict

File: service_catalog/mail_utils.py [hunk 2]
```diff
@@ -39,7 +39,7 @@ def _apply_when_filter_instance(user_qs, squest_object):
     user_emails = list()
     user_qs = _exclude_user_without_email(user_qs)
     if user_qs.filter(profile__instance_notification_enabled=False).exists():
-        logger.warning(f'The following users have no instance notification enabled:\n'
+        logger.warning(f"The following users have disabled their instance notification. They won't be notified:"
```
Comment (Sispheor) [inline service_catalog/mail_utils.py:42]: log info

File: service_catalog/mail_utils.py [hunk 3]
```diff
@@ -51,7 +51,7 @@ def _apply_when_filter_request(user_qs, squest_object):
     user_emails = list()
     user_qs = _exclude_user_without_email(user_qs)
     if user_qs.filter(profile__request_notification_enabled=False).exists():
-        logger.warning(f'The following users have no request notification enabled:'
+        logger.warning(f"The following users have disabled their request notification. They won't be notified:"
```
Comment (Sispheor) [inline service_catalog/mail_utils.py:54]: log info

File: service_catalog/views/operation.py [hunk 4]
```diff
@@ -84,7 +87,11 @@ def get_context_data(self, **kwargs):
         ]
         context['extra_html_button_path'] = "service_catalog/buttons/operation_survey_button.html"
         if self.request.user.has_perm('service_catalog.view_approvalworkflow'):
-            context['workflows_table'] = ApprovalWorkflowTable(ApprovalWorkflow.objects.filter(operation=self.get_object()),exclude=['operation'])
+            context['workflows_table'] = ApprovalWorkflowTable(
+                ApprovalWorkflow.objects.filter(operation=self.get_object()), exclude=['operation'],
+                prefix="operation-")
+            config.configure(context['workflows_table'])
```
Comment (EliasBoulharts) [inline service_catalog/views/operation.py:93]: only one table

File: service_catalog/views/tower_server.py [hunk 5]
```diff
@@ -26,9 +27,13 @@ class TowerServerDetailView(SquestDetailView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context["jobtemplate_table"] = JobTemplateTable(self.object.jobtemplate_set.all())
+        config = RequestConfig(self.request)
+        context["jobtemplate_table"] = JobTemplateTable(self.object.jobtemplate_set.all(), prefix="jobtemplate-")
+        config.configure(context['jobtemplate_table'])
```
Comment (EliasBoulharts) [inline service_catalog/views/tower_server.py:32]: one table

**Pull request merged**